_data/people.yaml

```diff
@@ -186,7 +185,6 @@ agossoubido2021:
   first-name: Agossou
   last-name: Bidossessi Emmanuel
   pronouns: He/Him/His
-  website: https://http://bidossessi.atwebpages.com/
```

Comment:
You can keep the website here. Just remove the extra `http://`

Reply:
Does this mean i am doing something right?

Reply:
I found a broken link on @yochannah `people.html`   https://www.codeisscience.com/manifesto  do I delete them? because it's flagged down as an empty link by [WAVE]{https://wave.webaim.org/}

Reply:
The correct link seems to be https://codeisscience.github.io/manifesto/

Reply:
I switched the link @bebatut  and it's working fine. You didn't actually give a reply to the question I asked days ago when I asked if I was on the right path by deleting the links I can't change

Reply:
I forgot that question. It would have been nice to add in the Pull Request description what are your issues
I think it would be good yes to delete the links that are not working

Reply:
i think i succeed in fixing some broken links @bebatut